### DISCUSSION TITLE
Panic during decryption

### DISCUSSION THREAD

**Discussion by BobWs** [navidrome/navidrome#3571] [General]
After updating my conainter to the latest version I see this in the log. How can I resolve this issue?
````
time="2024-12-21T13:29:51Z" level=error msg="Panic during decryption" error="runtime error: slice bounds out of range [12:6]"
time="2024-12-21T13:29:51Z" level=error msg="Could not decrypt JWT secret, creating a new one" error="decryption panicked"
````

**Comment by deluan** [comment]
This message is not clear (my bad), but it just mean that it could not find a encrypted JWT secret in the DB and it will create one. You should not see this message again after the upgrade.
